client/src/Landing.jsx

```diff
@@ -1,9 +1,19 @@
 import Example from "./Landing/Example";
 import How from "./Landing/How";
+import Modal from "./Landing/Modal";
+import React, { useState } from "react";
+import Header from "./Header";
 
 const Landing = () => {
+  const [openModal, setOpenModal] = useState(false);
+
+  const handleToggleModal = () => {
+    setOpenModal(!openModal);
+  };
   return (
     <>
+      <Header onToggleModal={handleToggleModal} />
```

Comment:
Header should not be rendered from within Landing. Reset to where Header is rendered in App (as is originally done) and have the modal code be moved to to Header component or somewhere else where it's independent or the context makes sense. 

That is, you want an image inside Header to render the modal, so the code could make sense in header. Or try it out in App to where Modal is rendered under Header from there.

Landing has nothing to do with logging in, and with the additional pages being added soon, you'd end up repeating this code.